server_installation/topics/operator/keycloak-backup-cr.adoc

```diff
@@ -1,5 +1,7 @@
 
 [[_backup-cr]]
+Backup CR is *deprecated* and could be removed in future releases.
+
 === Scheduling database backups
```

Comment:
@jonathanvila Sorry, one more suggestion. After building the docs I realized this note is very subtle and almost looks like it belongs to the previous chapter:
![Screenshot 2021-11-01 at 14 15 46](https://user-images.githubusercontent.com/13906855/139677569-c872f401-7261-44a1-827f-e98709d08ccd.png)


Maybe we could do something like:
```suggestion
[[_backup-cr]]
=== Scheduling database backups

[WARNING]
====
Backup CR is *deprecated* and could be removed in future releases.
====
```

Reply:
OK